catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/ApiSwaggerTest.scala

```diff
@@ -91,24 +91,7 @@ class ApiSwaggerTest extends FunSpec with Matchers with fixtures.Server {
       .toString shouldBe "\"#/definitions/Work\""
   }
 
-  // Because the Swagger should be the same on every request, we cache
-  // requests for the Swagger to save the time cost of starting a new
-  // server for every test/request.
-  //
-  var cachedResponses = Map[String, JsonNode]()
```

Comment:
this was causing issues in this branch and as a general rule in tests we should avoid mutable variables unless relly necessary